backend/de.metas.async/src/main/java/de/metas/async/processor/impl/WorkpackageProcessorTask.java

```diff
@@ -554,8 +554,8 @@ private void markSkipped(
 		}
 		final String msg = StringUtils.formatMessage("Skipped while processing workpackage by processor {}; workpackage={}", processorName, workPackage);
 
-		// log error (for later audit), but note that this exception can actually be quite frequent. so in order to not clutter the log, just log it on debug level
-		logger.debug(msg, skipException);
+		// switch to log debug if this exception turns out too frequently (to not clutter the log)
+		logger.error(msg, skipException);
```

Comment:
```suggestion
		logger.info(msg, skipException);
```

changing the level to info since it's not generally an error